app/src/main/java/com/openlattice/chronicle/services/notifications/NotificationsService.kt

```diff
@@ -0,0 +1,81 @@
+package com.openlattice.chronicle.services.notifications
+
+import android.app.*
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import com.openlattice.chronicle.R
+import com.openlattice.chronicle.preferences.EnrollmentSettings
+import com.openlattice.chronicle.receivers.lifecycle.NotificationsReceiver
+import java.util.*
+
+const val CHANNEL_ID = "Chronicle"
+
+class NotificationsService :IntentService ("SurveyNotificationsService") {
+
+    override fun onHandleIntent(intent: Intent) {
+        val enrollmentSettings = EnrollmentSettings(this)
+
+        if (enrollmentSettings.getNotificationsEnabled()) {
+            scheduleNotifications()
+        } else {
+            cancelPendingNotifications()
+        }
+    }
+
+
+    // schedule daily notifications at 7.00pm central time
+    // ref: https://developer.android.com/training/scheduling/alarms
+    private fun scheduleNotifications () {
+
+        val alarmManager : AlarmManager = getSystemService(Context.ALARM_SERVICE) as AlarmManager
+        val alarmIntent = createPendingAlarmIntent()
+
+        // set alarm to fire at 7.00pm central time
+        val calendar : Calendar = Calendar.getInstance()
+        calendar.set(Calendar.HOUR_OF_DAY, 19)
+        calendar.set(Calendar.MINUTE, 0)
+        calendar.timeZone = TimeZone.getTimeZone("Central Standard Time")
+
+        alarmManager.setInexactRepeating(AlarmManager.RTC_WAKEUP, calendar.timeInMillis, AlarmManager.INTERVAL_DAY, alarmIntent)
+    }
```

Comment:
What about users not in central time? 7pm is 5pm on the West Coast, and way past midnight in Europe.

Reply:
Good point. Perhaps we should do 7pm in all timezones?